locksmith/src/routes/hook.ts

```diff
@@ -0,0 +1,9 @@
+import express from 'express'
+import { subscriptionHandler } from '../controllers/hookController'
+
+const router = express.Router({ mergeParams: true })
+
+router.post('/:network/locks', subscriptionHandler)
```

Comment:
It might be worth adding the get version just for the sake of adding support to the websub discovery...
Other than that, lgtm!

Reply:
wouldn't that be handled by a publisher endpoint which advertises all our hubs and the urls?

Reply:
Will do!